docs/CONTRIBUTING.md

```diff
@@ -357,14 +380,51 @@ You can run individual unit tests by using the following syntax:
 Or by adding `.only` after a `describe` or `it` block in the test file:
 
 ```typescript
-describe.only('Extension manifest tests', () => {
-  it('should register all commands', () => {
+describe.only("Extension manifest tests", () => {
+  it("should register all commands", () => {
     // test code
   });
 });
 ```
 
-Remember to remove `.only` after running the test and before making a PR! 
+Remember to remove `.only` after running the test and before making a PR!
+
+#### Clicktesting the VS Code extension against the sidecar
+
+##### Method 1:
+
+Manually trigger the "Multi Arch Builds and Upload to GitHub Releases" promotion in the sidecar
+repo. This can be found in the Semaphore UI under the "Promotions" tab. This won't actually upload
+to a release, but will make the native executables available at the workflow's artifacts/ part of
+the Semaphore UI. Then you can continue on from step 3 of Method 2.
+
+##### Method 2:
+
+1. Checkout branch you want to test against in the sidecar.
+
+2. Run `make clean mvn-package-native-no-tests` in the sidecar to get the native executable packaged
```

Comment:
For easier copying and pasting:
```suggestion
2. Within the cloned `ide-sidecar` directory, run the following `make` command to build the native executable.
```
make clean mvn-package-native-no-tests
```
```

Reply:
nit: After checking out the Git repo and before running `make clean mvn-package-native-no-tests`, users should make sure they've installed the [prerequisites](https://github.com/confluentinc/ide-sidecar?tab=readme-ov-file#prerequisites-for-development).